app/src/test/java/com/example/android/architecture/blueprints/todoapp/LiveDataTestUtil.kt

```diff
@@ -63,7 +63,7 @@ fun <T> LiveData<T>.getOrAwaitValue(
 /**
  * Observes a [LiveData] until the `block` is done executing.
  */
-fun <T> LiveData<T>.observeForTesting(block: () -> Unit) {
+suspend fun <T> LiveData<T>.observeForTesting(block: suspend () -> Unit) {
```

Comment:
Revert, LiveData in this case has nothing to do with suspend funs and this class is copypasted a lot.

Reply:
Good point, done. This turned out to be easier than expected to work around.